client/src/modules/payroll/weekend_configuration/configuration.service.js

```diff
@@ -0,0 +1,32 @@
+angular.module('bhima.services')
+  .service('ConfigurationWeekEndService', ConfigurationWeekEndService);
+
+ConfigurationWeekEndService.$inject = ['PrototypeApiService', '$uibModal', '$http', 'util'];
+
+/**
+ * @class ConfigurationWeekEndService
+ * @extends PrototypeApiService
+ *
+ * @description
+ * Encapsulates common requests to the /weekend_config/ URL.
+ */
+function ConfigurationWeekEndService(Api, Modal, $http, util) {
+  var service = new Api('/weekend_config/');
```

Comment:
Just FYI: the `PrototypeApiService` embeds the `$http` and `util` parameters.  You can get them via `API.$http` or `service.$http` for both.